### PR TITLE
enh(monitoring/discovery): update doc to match 20.10 serie 

### PR DESCRIPTION
**Pull request by cgagnaire**

### CODE REVIEW

File: en/monitoring/discovery/administration.md [hunk 1]
```diff
@@ -98,133 +106,46 @@ and Remote Server or Pollers.
...
-
-1;
-```
-
-> The *clapi_user* and *clapi_password* directives might be changed to adapt to
-> an *admin* user crendentials needed for services creation and configuration
-> reloading.
-
-### Distributed architecture
+> If you had change the legacy *crond* configuration file to adapt the schedule
```
Comment (tuntoja) [inline en/monitoring/discovery/administration.md:124]: If you had changed the legacy *crond* configuration file to adapt the schedule

Comment (cgagnaire) [inline en/monitoring/discovery/administration.md:124]:
```suggestion
> If you had changed the legacy *crond* configuration file to adapt the schedule
```

**Review by lpinsivy — APPROVED** [review]

File: fr/monitoring/discovery/introduction.md [hunk 2]
```diff
@@ -34,8 +34,8 @@ automatiquement. Les nouvelles ressources découvertes automatiquement peuvent
 éventuellement être configurées automatiquement dans Centreon.
 
 Le même mécanisme de découverte peut s’appliquer à des ressources qui ne
-seraient plus disponibles. Ils peuvent être répertoriés et éventuellement
-désactivés à partir de la configuration.
+seraient plus disponibles. Elees peuvent être répertoriées et éventuellement
```
Comment (Thebarda) [inline fr/monitoring/discovery/introduction.md:37]:
```suggestion
seraient plus disponibles. Elles peuvent être répertoriées et éventuellement
```

File: fr/monitoring/discovery/hosts-discovery.md [hunk 3]
```diff
@@ -51,11 +61,74 @@ pourrait ressembler le résultat de la découverte :
...
+
+La dernière étape permet de choisir parmis deux méthodes d'exécution.
+
+#### Exécuter immédiatement
+
+L'exécution immédiate lancera la découverte juste après la création de la tâche.
+
+#### Planifier l'exécution
+
+L'exécution planifiée permet de choisir parmis plusieurs types de planification
```
Comment (Thebarda) [inline fr/monitoring/discovery/hosts-discovery.md:106]:
```suggestion
L'exécution planifiée permet de choisir parmi plusieurs types de planification
```

**Review by Thebarda — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**